server/models.py

```diff
@@ -715,6 +715,10 @@ def create(cid, enrollment_info=None, role=STUDENT_ROLE):
 
         cache.delete_memoized(User.is_enrolled)
 
+    @staticmethod
+    def get_all_students(cid):
+        return Enrollment.query.filter_by(course_id=cid, role=STUDENT_ROLE).all()
```

Comment:
Because of this line 
```
class Enrollment 
    ... 
    course = db.relationship("Course", backref="participations")
``` 

We can access all enrollments by simply doing `course.participations`. It doesn't filter for only students - but you can do that in the API endpoint.